.github/workflows/golangci-lint.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v1.29
+          version: latest
```

Comment:
NIT: Not sure it is a good idea. I am afraid that a future version may expect a newer Go version or something. Or it is possible that the lint checks will start catching new problems and the pipeline will start to fail.

Reply:
Changed to latest available version